dashboard/src/pages/hackathon/Register.vue

```
@@ -20,6 +20,7 @@
       <img
         v-if="hackathon.data.hackathon_logo"
         :src="hackathon.data.hackathon_logo"
+        :alt="hackathon.data.hackathon_name ? `Logo of ${hackathon.data.hackathon_name}` : 'Hackathon logo'"
```

ESLint

[error] Replace `hackathon.data.hackathon_name·?·`Logo·of·${hackathon.data.hackathon_name}`·:·'Hackathon·logo'` with `⏎··········hackathon.data.hackathon_name⏎············?·`Logo·of·${hackathon.data.hackathon_name}`⏎············:·'Hackathon·logo'⏎········`

(prettier/prettier)